src/VRCFaceTracking.Avalonia/App.axaml

```diff
@@ -39,7 +40,7 @@
                     ChromeGray="#ff737373"
                     ChromeHigh="#ffcccccc"
                     ChromeLow="#ffececec"
-                    ChromeMedium="#ffe6e6e6"
+                    ChromeMedium="#ffdbdbdb"
```

Comment:
Looking at the design, I think the solution here is to go _brighter_ instead of darker. Try just plain "White" here, it provides the contrast we're looking for.

Reply:
We could, as an alternative, just pick a different color to use in the background instead of using `ChromeMedium`.

Reply:
Making ChromeMedium white works fine for me.